### DISCUSSION TITLE
Can I run `plotman plots` and `plotman interactive`

### DISCUSSION THREAD

**Discussion by mcybz** [ericaltendorf/plotman#407] [Q&A]
Can multiple plotmans run at once?

Basically I want start a bunch of headless plotmans in the background on different machines via SSH using `plotman plot &` 

However I also now and then check on plotman by running `plotman interactive`. When I do that however two plotmans will technically be active. Is this OK? Is my overall approach what others are doing?

**Comment by Amrac** [comment]
I think it is ok, I did it without issues. 

If you use the archive function, please note that plotman plot won't archive. 
If you change your configuration, you need to stop both plotman plot and plotman interactive

**Reply by altendky** [reply]
In the past there was a hazard (that did happen) that the two instances would launch a plot at the same time and both plotting processes would log to one file which was...  confusing.  They won't do that anymore but can still launch two plots in rapid succession.  Once `interactive` is running, you can press `p` to disable the plot launching (and similarly `a` for archiving).  There is PR to add a command line option for this.  Longer term, I do want to make plotman lend itself more directly to this usage.  For now, you can do it with some annoyance and caveats.
